app/views/file_imports/edit.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Edit <%= @file_import.active_storage_attachment.blob.filename %></h2>
```

Comment:
How about making an instance method on the FileImport model that returns the filename since we are trying to access this somewhat frequently?